roiextractors/segmentationextractor.py

```diff
@@ -205,6 +208,28 @@ def get_image_size(self) -> ArrayType:
         """
         pass
 
+    @abstractmethod
+    def get_traces(self) -> ArrayType:
```

Comment:
how would a user access traces for Neuropil or Background? What about specific indices or channels?

Reply:
corrected here e3a438f6b82bf8aa40e91dbfee1f1e4f709f689e